public/web-app/cypress/e2e/connectez-vous-tests.cy.js

```diff
@@ -1,9 +1,11 @@
+//npx cypress open  
+
 // test connecte client
 describe('template spec', () => {
   it('passes', () => {
```

Comment:
Par exemple :
describe('Tests de connexion de l’utilisateur', () => {
it('devrait permettre à un utilisateur de se connecter via le menu', () => {

Renommer aussi connecte-tests.cy.js